docs/webhooks.rst

```diff
@@ -36,6 +36,8 @@ As an example, the URL pattern looks like this: *readthedocs.org/api/v2/webhook/
 
 Use this URL when setting up a new webhook with your provider -- these steps vary depending on the provider:
 
+.. _webhook-integration-github:
```

Comment:
We are using http://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html to avoid doing this

Reply:
ah yes. I added these as I wanted to point from our UI to these -- meaning the headings can't be allowed to change. Perhaps explicit labels is most helpful here? Also doc comments could warn doc authors though.

Reply:
I think we should _only_ use explicit when we are linking them from another place and avoid adding them "just because" or "just in case we need it later".

That way, we will know that if it's there it's because an explicit reason.

Reply:
I think the linter or sphinx itself can catch errors if the title is renamed. And probably is worth adding this to the docs guide https://docs.readthedocs.io/en/latest/docs.html

Reply:
If this is catch automatically, I'd love it! I'm :+1: on setting this up.

Reply:
We can't turn on the sphinx check yet in the whole project bc of https://github.com/rtfd/readthedocs.org/pull/4166#issuecomment-393382331 what I usually do is just check the warnings locally :)

Reply:
This is more than I want to tackle now. I'll leave them and we can revisit later to either throw errors or something. I don't think removing is important right now.